.circleci/workflows.yml

```diff
@@ -2036,7 +2033,7 @@ jobs:
           command: rm -rf cypress.json
       - run:
           name: Install prod dependencies
-          command: yarn --production
+          command: yarn --production --ignore-engines
```

Comment:
since you are updating this, we should be able to remove the `rename support file` step since v10 has been released for a while now and this was the ci mitigation while that was in flight